### PR TITLE
Update /download/iot with new section and link tags

### PR DESCRIPTION
**Pull request by petesfrench**
## Done
**To go live on June 16th, see doc for more details**
- Update /download/iot with new section and link tags

## QA

- Go to https://ubuntu-com-11678.demos.haus/download/iot and compare against [copydoc](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit#heading=h.rcu78t62sv8b)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5423

### CODE REVIEW

**Comment by petesfrench** [comment]
@spencerbygraves There is a request in the [copydoc](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit#heading=h.rcu78t62sv8b) to have a different background colour on the section 'Ubuntu Core 22 Beta is now available'. But I am not sure how that will work with the current flow of the page, any suggestions?

**Comment by spencerbygraves** [comment]
@petesfrench perhaps you could try splitting that section into 2 strips, and trying `p-strip--suru` on the beta one.

**Comment by petesfrench** [comment]
Thank you @spencerbygraves !

File: templates/download/iot/index.html [hunk 1]
```diff
@@ -115,6 +115,23 @@ <h3>Supported platforms</h3>
         </p>
       </div>
     </div>
+    <div class="row">
+      <div class="col-8">
+        <p>Ubuntu works along with silicon vendors and manufacturers to enable and certify Ubuntu on a wide range of hardware. The certified hardware passes our extensive testing and review process in our labs, ensuring that Ubuntu runs well out of the box, and regression tests are performed before every system update is delivered.</p>
+        <p><a href="https://ubuntu.com/certified" class="p-button">Browse the full list of certified hardware</a></p>
```
Comment (mtruj013) [inline templates/download/iot/index.html:121]: David made a change on the copy doc and seems to want this button above the paragraph which is not our usual convention. Maybe that's a design or ux issue? He also changed the text to "The ultimate Ubuntu Experience: Get Certified Hardware"

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good @petesfrench! There's been a change on the doc that needs addressing and also there's a spacing bug on line 32

**Comment by petesfrench** [comment]
Closing as this is being completed in another [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/11699)

**Pull request closed without merging**